markdown/guide/what-are-engines.md

```diff
@@ -1,27 +1,31 @@
+Even though the application has sets of functionality represented by Engines, there is room for some additional functionality that the host application itself provides.
 
-The oft used example of this is a "blogging" application. While a blog itself could be a full-fledged application, you may also wish to use it as a specific subsection of your overall application. If you threw a "news feed" engine into that mix, you might get an application structure that looks something like this:
+## Why use Engines?
 
-![Diagram of Application with two Engines](../images/02-app-diagram.png)
+Large companies are increasingly adopting Ember.js to power their entire product lines. Often this means separate teams (sometimes distributed around the world) working on the same app. Typically, responsibility is shared by dividing the application into one or more "sections". How this division is actually implemented varies from team to team. 
 
-Even though the application has sets of functionality represented by Engines, there is room for some additional functionality that the host application itself provides.
+These companies also have large monolithic applications, which pose the following challenges:
 
-### How Engines Differ From Applications
+* `Side effects` - if you change something, it may be unclear how it could affect the rest of platform.
+* `Coordination` - when you develop a new feature or make big changes, many teams may need to be in sync to approve it.
+* `Complexity` - with a huge dependency tree and many layers of abstraction, developers cannot iterate quickly, and features suffer as a result.
+* `Killing Innovation` - a/b testing a cutting-edge feature is hard to do without disrupting the rest of the app and the teams working on it.
+* `Slow Onboarding` - new people coming into the team are overwhelmed.
 
-Now that we can understand what Engines represent conceptually (that is, "logical applications"), let's talk about the concrete differences between Applications and Engines. Again, from the Engines RFC:
+Engines provide an alternative to these approaches that allows for distributed development, testing, and packaging. The goal of Ember Engines was to allow large Ember applications to be split into consumable Addons allowing development teams to build logically-grouped pieces of an application.
 
-> The primary differences [between Applications and Engines] are that an engine does not boot itself and an engine does not control the router.
+Engines are good for companies that have a large number of teams, where each team has their own area that is clearly separated from the others. If your app is developed by a small team, engines are probably not very helpful for you. The isolation is good when it matches organizational boundaries, but adds unnecessary complexity when there is not a good match.
```

Comment:
I find this paragraph overly prescriptive and would probably just remove it. Who are we to say that engines couldn't be useful for two different developers working independently on a single application?

Reply:
@dgeb this is the same info on official ember guides approved by core team - https://guides.emberjs.com/release/applications/ember-engines/

And we're just saying it's probably not good, we're not giving a certainty, but almost always this seems to be overengineering - "engines are probably not very helpful for you."

Reply:
@villander ok, I agree they should be consistent, but I really think this should be toned down in both places.  I feel that this recommendation is heavy handed. Saying "large number of teams" is overboard. It's fine to say multiple teams. And sometimes those are "teams" of one. The point being that Engines are an organizational solution.

Reply:
And we need to say "organizations" and not "companies".

Reply:
makes sense, I'll change it here, and on official guides on the future.

Reply:
fixed @dgeb